### PR TITLE
Feature settlement placement

### PR DESCRIPTION
**Pull request by Danimineiro**

### CODE REVIEW

File: Source/1.3/Windows/SettlementPlacementWindow.cs [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+            }
+
+            return flag;
+        }
+
+        public void ApplyAction()
+        {
+            if (!CanPlaceHere(out List<string> reasons, playerEmpire))
+            {
+                Messages.Message(new Message(string.Join("\n", reasons), MessageTypeDefOf.RejectInput));
```
Comment (Toby222) [inline Source/1.3/Windows/SettlementPlacementWindow.cs]: Multiline Messages look weird and overlap on the screen.
Consider replacing with a popup window, maybe?

Comment (Danimineiro) [reply Source/1.3/Windows/SettlementPlacementWindow.cs]: I forgor

Comment (Danimineiro) [reply Source/1.3/Windows/SettlementPlacementWindow.cs]: [17d3c62](https://github.com/Danimineiro/Empire-Rewritten/pull/30/commits/17d3c6267516fc359cf929b7db61858461b127a5) fixes

**Comment by Danimineiro** [comment]
The empty part at the bottom is intentionally empty for potential stuff after we got the delivery system

**Review by RecompiledBirds — APPROVED** [review]

**Pull request merged**